scripts/install.sh

```diff
@@ -1,49 +1,94 @@
 #!/bin/bash
-set -e
+set -euo pipefail
+
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
 
 # Dafault inputs.
-KEEP_ETHEREUM_PASSWORD_DEFAULT="password"
 KEEP_CORE_PATH=$PWD
+KEEP_CORE_SOL_PATH="$KEEP_CORE_PATH/solidity"
 CONFIG_DIR_PATH_DEFAULT="$KEEP_CORE_PATH/configs"
+NETWORK_DEFAULT="local"
+KEEP_ACCOUNT_PASSWORD=${KEEP_HOST_CHAIN_ACCOUNT_PASSWORD:-"password"}
```

Comment:
Right, but in case of `CONFIG_DIR_PATH_DEFAULT` & `NETWORK_DEFAULT` we are using them later in the code where focus is not on inputs like in the "Default inputs" block of this script. If we introduce a separate `KEEP_ETHEREUM_PASSWORD_DEFAULT` it will only be used in the "Default inputs" block: KEEP_ACCOUNT_PASSWORD=${KEEP_HOST_CHAIN_ACCOUNT_PASSWORD:-$KEEP_ETHEREUM_PASSWORD_DEFAULT} and we have to add a new var, which at this point will grow password related vars to 3 to describe 1 final password. 

The same concerns `ACCOUNT_PRIVATE_KEY=${CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY:-""}` which takes an empty string for default.

Anyway, I'm actually okay with both and see pros and cons. We can introduce 2 new vars: `KEEP_ACCOUNT_PASSWORD_DEFAULT` + `CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY_DEFAULT` or I can add a comment next to `KEEP_ACCOUNT_PASSWORD=${KEEP_HOST_CHAIN_ACCOUNT_PASSWORD:-"password"}` saying "password" is a default password & same for the private key.

Reply:
On second thought, if we want to stick to `KEEP_ETHEREUM_PASSWORD` per the [previous comment](https://github.com/keep-network/keep-core/pull/2309#discussion_r572106225) and we want to accept it as an environment variable, we do not need `KEEP_ETHEREUM_PASSWORD_DEFAULT` at all. We should just expect it is supplied. 

Side note, we should probably move `KEEP_CORE_PATH` and `KEEP_CORE_SOL_PATH` out of this section, those are not defaults:
```
KEEP_CORE_PATH=$PWD
KEEP_CORE_SOL_PATH="$KEEP_CORE_PATH/solidity"

# Defaults, can be overwritten by env variables/input parameters
CONFIG_DIR_PATH_DEFAULT="$KEEP_CORE_PATH/configs"
NETWORK_DEFAULT="local"
```

Reply:
56aa8f1

```
KEEP_CORE_PATH=$PWD
KEEP_CORE_SOL_PATH="$KEEP_CORE_PATH/solidity"

# Defaults, can be overwritten by env variables/input parameters
CONFIG_DIR_PATH_DEFAULT="$KEEP_CORE_PATH/configs"
NETWORK_DEFAULT="local"
KEEP_ETHEREUM_PASSWORD=${KEEP_ETH_ACCOUNT_PASSWORD:-"password"}
CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY=${CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY:-""}
```
We will need to include `KEEP_ETHEREUM_PASSWORD` & `CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY` because of `set -u`. Some more reasoning behind added options are [here](https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/)